app/src/main/java/net/gsantner/markor/format/keyvalue/KeyValueHighlighter.java

```diff
@@ -11,7 +11,7 @@
 
 import android.content.Context;
 import android.graphics.Typeface;
-import android.text.Editable;
```

Comment:
The only change in `MarkdownHighlighter.java`, `KeyValueHighlighter.java`, `PlainTextHighlighter.java` is Editable -> Spannable